doc/release-notes-33106.md

```diff
@@ -0,0 +1,17 @@
+Mining and Transaction Relay Policy
+=========================
+
+The minimum block feerate (`-blockmintxfee`) has been changed to 1 satoshi per kvB. It can still be changed using the
+configuration option.
+
+The default minimum relay feerate (`-minrelaytxfee`) and incremental relay feerate (`-incrementalrelayfee`) have been
+changed to 100 satoshis per kvB. They can still be changed using their respective configuration options, but it is
```

Comment:
Would be nice to mention in the release notes the original values these constants have been changed from.